spec/datacite_utils_spec.rb

```diff
@@ -175,4 +175,40 @@
       expect(response.dig("descriptions", "description")).to eq("descriptionType" => "Abstract", "__content__" => "Eating your own dog food is a slang term to describe that an organization should itself use the products and services it provides. For DataCite this means that we should use DOIs with appropriate metadata and strategies for long-term preservation for...")
     end
   end
+
+  context "insert_related_items" do
+    it "insert" do
+      # input = "spec/fixtures/datacite-example-relateditems.xml"
+      input = "https://handle.stage.datacite.org/10.70048/290320201351"
```

Comment:
The fixture was just my first pass at the test, because I didn't have an example to hand from production that had related items. Once I realised I could use a DOI from staging and a VCR, I changed it to bring it line with most of the other tests, which use
```ruby
  let(:input) { "https://doi.org/10.5061/DRYAD.8515" }

  subject { Bolognese::Metadata.new(input: input, from: "datacite") }
```
from the top of the file.

I could change it back to the fixture without any issue if that is preferred.

Reply:
I say without issue, it broke the test in a way that doesn't make any sense :grimacing:

Reply:
Not enough coffee error :coffee:

Reply:
That's a live DOI though so a little more stable, there are no guarantees with handles on stage. 
Curiously I had thought most of them were using fixtures not the DOI and VCR, but it seems to be mixed.
I prefer fixtures here for the bolognese library than I do to relying on VCR, but I could be convinced otherwise.